docker-build-env/Dockerfile

```diff
@@ -10,20 +10,11 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get -y --no-install-rec
         git \
         googletest \
         libboost-all-dev \
-        # libp11 engine
-        libengine-pkcs11-openssl \
-        # headers for p11 engine
-        # TODO: this may not be necessary if we don't use libp11 directly (only though openssl)
-        libp11-dev \
-        # softhsm2 module
-        libsofthsm2 \
         libssl-dev \
+        # For llvm-symbolizer (used by ASAN).
+        llvm \
```

Comment:
Any specific reason why we need this now?

Reply:
No, I can push this to another PR. Fixing.